krabs/krabs/ut.hpp

```diff
@@ -19,6 +19,14 @@ namespace krabs { namespace details {
     struct ut {
 
         typedef krabs::provider<> provider_type;
+		
+		struct FilterSettings
+		{
+			std::vector<unsigned short> m_OrigEventIds;
+			std::tuple<UCHAR, ULONGLONG, ULONGLONG, UCHAR> flagsTuple;
```

Comment:
Please use snake_case (`flags_tuple_`):
```suggestion
			std::tuple<UCHAR, ULONGLONG, ULONGLONG, UCHAR> flags_tuple_;
```

Reply:
Done